services/supabase/migrations/20260722224400_add_profiles_pollar_fields.sql

```
@@ -0,0 +1,24 @@
+ALTER TABLE public.profiles
+  ADD COLUMN IF NOT EXISTS onboarding_provider public.onboarding_provider NOT NULL DEFAULT 'legacy_passkey',
+  ADD COLUMN IF NOT EXISTS pollar_user_id text,
+  ADD COLUMN IF NOT EXISTS pollar_wallet_address text,
+  ADD COLUMN IF NOT EXISTS pollar_wallet_activated_at timestamptz;
+
+CREATE UNIQUE INDEX IF NOT EXISTS profiles_pollar_user_id_key
+  ON public.profiles (pollar_user_id)
+  WHERE pollar_user_id IS NOT NULL;
+
+CREATE INDEX IF NOT EXISTS idx_profiles_onboarding_provider
+  ON public.profiles (onboarding_provider);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)